examples/neighbors/plot_lmnn_classification.py

```diff
@@ -0,0 +1,80 @@
+"""
+============================================
+Large Margin Nearest Neighbor Classification
+============================================
+
+Sample usage of Large Margin Nearest Neighbor classification.
+It will plot the decision boundaries for each class.
```

Comment:
Can say a tiny bit more here, so that a user who Googles and lands on this page understands better why and what.

Reply:
Done.